base/math.jl

```diff
@@ -404,6 +406,28 @@ cosh(x::Number)
     tanh(x)
 
 Compute hyperbolic tangent of `x`.
+
+See also [`tan`](@ref), [`atanh`](@ref).
+
+# Examples
+
+```jldoctest
+julia> tanh.(-3:3f0)
```

Comment:
It's mostly just a hack to avoid printing 16 digits here. 

But also, strikes me as easy to ignore if you don't know, and something which you eventually should know. If you paste that bit into the REPL to see, then I guess you don't learn much:
```
julia> -3:3f0
-3.0f0:1.0f0:3.0f0
```
Maybe it could be clarified in some other way?

Reply:
Might be better to use the explicit `round` broadcasting like you did for `sin`. Or else just live with the 16 digits of printing. Doing the rounding feels kind of like hiding the real output anyways.